### PR TITLE
DOC Document SecurityAdmin extending ModelAdmin

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1350

### CODE REVIEW

File: en/04_Changelogs/5.0.0.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ title: 5.0.0 (unreleased)
 
 This is a major release and as a result there are a number of breaking API changes. For a full list of these see [upgrading your project](/upgrading/upgrading_your_project). Some specific details about a few of them are below.
 
+- SecurityAdmin now extends ModelAdmin instead of LeftAndMain. This was done to remove duplicate code.
```
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.0.0.md]: Probably want more details in here than that - e.g. some paths will be changing, extension hooks are likely to have changed, etc.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

**Comment by GuySartorelli** [comment]
Rebased on `5` as a part of https://github.com/silverstripe/developer-docs/issues/74 - but the review still stands.

File: en/04_Changelogs/5.0.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/04_Changelogs/5.0.0.md]: Updated.  I don't think the changes in path are probably too much details as they're more of a UX thing.

Comment (ChloeSartorelli) [reply en/04_Changelogs/5.0.0.md]: The changes in paths may have implications for customisations that call custom actions in extensions, for example. It's worth highlighting IMO and there's no harm in doing so.

Comment (emteknetnz) [reply en/04_Changelogs/5.0.0.md]: Updated

File: en/04_Changelogs/5.0.0.md [hunk 2]
```diff
@@ -82,6 +82,11 @@ This is a major release and as a result there are a number of breaking API chang
 
 ### General changes {#api-general}
 
+- [SecurityAdmin](api:SilverStripe\Admin\SecurityAdmin) now extends [ModelAdmin](api:SilverStripe\Admin\ModelAdmin) instead of [LeftAndMain](api:SilverStripe\Admin\LeftAndMain) so that it can make use of standard API. This was done to remove duplicate code.
+  - `MemberImportForm.php` has been removed
+  - `GroupImportForm.php` has been removed
+  - Previous extension hook implementations of `SecurityAdmin::updateEditForm($form)` should still work as the extension hook is called in `ModelAdmin::getEditForm()`
```
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.0.0.md:88]: The actual form is slightly different so the extensions may need to be reworked slightly to work with the way modeladmin's form is produced.... but we can work out the exact wording for that closer to release. For now this is good enough.

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**